### DISCUSSION TITLE
Q: Disable comment counter

### DISCUSSION THREAD

**Discussion by githubhabIT** [walinejs/waline#2532] [Q&A] [answered]
### Describe the bug | 问题描述

How do I turn off the comment count counter?
![image](https://github.com/walinejs/waline/assets/63256627/9feeecea-3d63-4281-8eef-cffc56f66168)

### Website URL | 问题网站

https://xboxrussia.tilda.ws/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

How do I turn off the comment count counter?
![image](https://github.com/walinejs/waline/assets/63256627/9feeecea-3d63-4281-8eef-cffc56f66168)

### Website URL | Problem website

https://xboxrussia.tilda.ws/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
You can hidden counter with custom css in your html, just add the css in your html:

```css
.wl-count { visibility: hidden; }
```

**Comment by githubhabIT** [comment]
Thank you 🔥
